tests/fast-integration/skr-test/README.md

```diff
@@ -2,16 +2,44 @@
 
 This test covers SKR (SAP Kyma Runtime).
 
-## File Structure consists of the following parts:
+## File structure
 - **provision** folder contains the scripts for provisioning and de-provisioning the SKR cluster using KEB client
-- **oidc** folder contains the OIDC related tests
+- **oidc** folder contains the OIDC-related tests
 - **commerce-mock** folder consists of two scripts:
-  1. **index.js** contains the commerce mock tests suite
-  2. **prep.js** can be called in stand-alone mode for commerce mock resources preparation
+  - **index.js** contains the commerce mock tests suite
+  - **prep.js** can be called in stand-alone mode for CommerceMock resources preparation
 
-##Run the SKR test
+## Usage modes
 
-Before you run the test make sure you prepare the `.env` file based on the `.env.template`:
+You can use the SKR test in two modes - with and without provisioning.
+
+### With provisioning
+
+In this mode, the test executes the following steps:
+
+1. Provision SKR cluster and register compass resources
+2. Run OIDC Test
+3. Run Commerce Mock Test
+4. De-provision the SKR instance and clean the compass resources.
+
+### Without Provisioning.
+
+In this mode the test will additionally needs the following environment variables:
+- `SKIP_PROVISIONING`, set to `true`
+- `INSTANCE_ID` the uuid of the provisioned SKR instance
+- `TEST_SUFFIX` determines the compass scenario and the app assigned to it. If not set, it's randomly generated.
+
+>**IMPORTANT:** The `TEST_SUFFIX must be 4 characters long and is required for every commerce-mock test run. In case the compass scenario with the given suffix already exists, the test will try to reuse it.
+
+In this mode, the test executes the following steps:
+1. Ensure SKR exists and register compass resources
+2. Run OIDC Test
+3. Run Commerce Mock Test
+4. Clean the compass resources
+
```

Comment:
Adding it here, so that users aren't surprised by the 3rd option later :D 

```suggestion

> **NOTE:** The SKR test additionally contains a stand-alone script, which you can use to register the Commerce Mock resources. 

```

Reply:
done